components/icons/DealsIcon.tsx

```diff
@@ -0,0 +1,10 @@
+export default function DealsIcon(props: any) {
+  return (
+    <svg {...props} width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" data-reactroot="">
```

Comment:
This needs to be fixed to be React compatible, `strokeLinejoin` `strokeLinecap`.. then for color use `currentColor` and let the CSS of the parent element dictate the color.